### DISCUSSION TITLE
Will windi lang require braces, colons and semicolons?

### DISCUSSION THREAD

**Discussion by wvffle** [windicss/windicss#378] [Ideas]
From what I can see on https://windicss.org/posts/language.html, you're designing the language to be similar to CSS when it comes to braces, colons and semicolons.

Personally, I fell in love with https://stylus-lang.com/ because those characters are optional. It saves a quite a lot of time when you do not write them. Also in my opinion, the code is much clearer
```stylus
h1, h2, h3
h4, h5, h6
  margin 0

  span
    display block
```
is the same as
```stylus
h1, h2, h3,
h4, h5, h6 {
  margin: 0;
}

h1 span, h2 span, h3 span,
h4 span, h5 span, h6 span {
  display: block;
}
```

My question is: will windi lang require braces, colons and semicolons?

**Comment by matthewjumpsoffbuildings** [comment]
yea i am very committed to the sass (not scss) indentation based syntax, which is quite similar to stylus

i really hope windi lang offers this kind of syntax as an option
